### PR TITLE
remove reference to System.Reflection.Emit for UAP builds of System.Linq.Expressions

### PR DESCRIPTION
**Pull request by VSadov**
remove reference to System.Reflection.Emit for UAP builds of System.Linq.Expressions

### CODE REVIEW

**Comment by VSadov** [comment]
Do not merge this just yet. I want to see if it passes CI and if fails how and why.

**Comment by VSadov** [comment]
I don't get it. Setting  "IsInterpreting" is supposed to exclude all the files that generate errors in uapaot. :-\

**Comment by VSadov** [comment]
And it does when I build locally

**Comment by VSadov** [comment]
I see. "uap" and "uapaot" make the difference

File: src/Common/tests/System/TypeBuilderExtensions.cs [hunk 1]
```diff
@@ -1,10 +1,14 @@
 using System;
 using System.Reflection;
+
+#if FEATURE_COMPILE
 using System.Reflection.Emit;
 
 // This is a temporary workaround for TypeBuilder and EnumBuilder not extending TypeInfo. See https://github.com/dotnet/corefx/issues/14334.
```
Comment (jkotas) [inline src/Common/tests/System/TypeBuilderExtensions.cs]: You can just delete this while you are on it.

**Comment by VSadov** [comment]
@jkotas - Thanks!  Deleted.

**Comment by VSadov** [comment]
@danmosemsft @joperezr @OmarTawfik @jaredpar @DnlHarvey   - FYI

**Comment by VSadov** [comment]
This should unblock building System.Linq.Expressons for  "uapaot" without references to Reflection.Emit

**Pull request merged**